packages/v4/patternfly-docs/content/design-guidelines/components/inline-edit/inline-edit.md

```diff
@@ -4,28 +4,33 @@ section: components
 ---
 
 An **inline edit** component allows users to switch between a read-only view and an edit view of [description lists](/components/description-list), page text elements, or [tables](/components/table) as needed. Inline editing saves the user from opening a modal window or redirecting to a separate edit page so that they can edit in context of their current view. 
+## Elements
+
+Elements vary depending on inline edit variation
+* [Page inline elements](#page-inline-elements)
+* [Table inline elements](#table-inline-elements)
 
 ## Usage
 
-Use inline edit when:
+### When to use
 * All editable elements can be viewed within the row or expanded row.
 * The data needs to be updated frequently.
- 
-Don’t use inline edit when:
+
+### When not to use
 * The editing is the primary function of the view. For example, in an edit modal.
 
 <img src="./img/inline-edit.gif" width="960" height="540" />.
```

Comment:
This whole GIF should probably be in a behavior section no? / A behavior section should be added for this one to describe the process because GIFs, we've realized, aren't very accessible